### ISSUE TITLE
copr build state: store up to date info in the DB

### ISSUE THREAD

**Issue by TomasTomecek** [packit-service/packit-service#517]
The COPR team deployed a change in prod that allows us to get build state per chroot. Let's implement this on our side as well:

https://bugzilla.redhat.com/show_bug.cgi?id=1813227

**Comment by rishavanand** [comment]
Hey @TomasTomecek 
I was looking into this and got the following error:
```
TypeError('get() takes 2 positional arguments but 3 were given')
```
when using this statement
```
ch = copr_client.build_proxy.get(build_id, "fedora-31-x86_64")
```
I also cross-checked `copr` package for the latest one. Do you know what problem might be?

**Comment by TomasTomecek** [comment]
@rishavanand thanks for looking into this! You're close: this issue is about updating code in tasks.py:

https://github.com/packit-service/packit-service/blob/master/packit_service/worker/tasks.py#L83

So just parse the `build_proxy.get` response:
```python
build_copr = copr_client.build_proxy.get(build_id)
```

**Comment by rishavanand** [comment]
@TomasTomecek well that's exactly what I did. I parsed the `build_proxy.get` response but did not find any status for chroot. So I went ahead to read the comments on bugzilla. According to this [comment #2](https://bugzilla.redhat.com/show_bug.cgi?id=1813227#c2), two params will have to be passed to the get method, but this leads to the error listed above. There is something mentioned related to a frontend release, maybe that is when we will be able to use this feature?
